packages/shared/friends/sagas.ts

```diff
@@ -537,14 +554,18 @@ function* refreshFriends() {
 
     const requestedFromIds = fromFriendRequests.map(
       (request): FriendRequest => ({
+        friendRequestId: `${getUserIdFromMatrix(request.from)}_${getUserIdFromMatrix(ownId)}`, // {from_requester}_{to_requested} -> 'otherUserId_ownId',
         createdAt: request.createdAt,
-        userId: getUserIdFromMatrix(request.from)
+        userId: getUserIdFromMatrix(request.from),
+        message: request.message
       })
     )
     const requestedToIds = toFriendRequests.map(
       (request): FriendRequest => ({
+        friendRequestId: `${getUserIdFromMatrix(ownId)}_${getUserIdFromMatrix(request.to)}`, // {from_requester}_{to_requested} -> 'ownId_otherUserId',
```

Comment:
I'm thinking it might be better to have the same criterion as for the friendship channel, i.e. `ownId < otherUserId ? ownId_otherUserId : otherUserId_ownId`. WDYT?
Also let's extract this pattern into a function to keep the same format everywhere we need to generate the request id.